examples/deployment-chain/simple/deployment.yaml

```diff
@@ -0,0 +1,27 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: simple
```

Comment:
Rename all those things to `chain-simple` to avoid conflicting with the simple app in the `kubernetes` directory.